### ISSUE TITLE
Appears to only work with display :0

### ISSUE THREAD

**Issue by amckee** [symless/synergy-core#6263]
Installed synergy_2.0.6.stable~b1396+1ef57bb6_amd64.deb on a fairly fresh installation of latest Kali:
> [root@tardis:/etc/synergy] cat /etc/*release
> DISTRIB_ID=Kali
> DISTRIB_RELEASE=kali-rolling
> DISTRIB_CODENAME=kali-rolling
> NAME="Kali GNU/Linux"
> ID=kali
> VERSION="2018.1"
> ID_LIKE=debian
> HOME_URL="http://www.kali.org/"

The logs are pretty clear, so I'll start with that (the conf file mentioned is unmodded by me):
/var/log/synergy/synergy-combined.log
```
[ Core    ] [2018-02-26T20:31:11] INFO: ignoring --enable-drag-drop, not supported on linux.
[ Core    ] [2018-02-26T20:31:11] DEBUG: opening configuration "/var/lib/synergy/synergy.conf"
[ Core    ] [2018-02-26T20:31:11] DEBUG: configuration read successfully
[ Core    ] [2018-02-26T20:31:11] DEBUG: XOpenDisplay(":0.0")
[ Core    ] No protocol specified
[ Core    ] [2018-02-26T20:31:11] WARNING: primary screen unavailable: unable to open screen
```

As you can see, my X display is running on :1
```
[root@tardis:/home/adam] w
 21:54:59 up 2 days,  9:46,  4 users,  load average: 3.14, 3.46, 3.21
USER     TTY      FROM             LOGIN@   IDLE   JCPU   PCPU WHAT
adam     :1       :1               Sat14   ?xdm?  51:10   0.01s /usr/lib/gdm3/gdm-x-session --run-script /usr/bin/openbox-session
```

Without man pages or a help dialog, I'm not sure what options to try.
```
[root@tardis:/etc/synergy] synergy-core --help
error: use --client or --server args
[root@tardis:/etc/synergy] man synergy-core
No manual entry for synergy-core

[root@tardis:/etc/synergy] synergy-core --version
error: use --client or --server args
```
I will add more log/configs/more details if you'd like, but it seems rather moot given the above.

**Comment by amckee** [comment]
Setting DISPLAY didn't work, as root or user:

```
[root@tardis:/etc/synergy] synergy-core --server -c /var/lib/synergy/synergy.conf
[2018-02-26T22:27:28] WARNING: failed to set process uid to: -1
[root@tardis:/etc/synergy] DISPLAY=:1 synergy-core --server -c /var/lib/synergy/synergy.conf
[2018-02-26T22:53:10] WARNING: failed to set process uid to: -1

[adam@tardis:/home/adam] synergy-core --server -c /var/lib/synergy/synergy.conf
[2018-02-26T22:27:36] WARNING: failed to set process uid to: -1
[adam@tardis:/home/adam] DISPLAY=:1 synergy-core --server -c /var/lib/synergy/synergy.conf
[2018-02-26T22:53:01] WARNING: failed to set process uid to: -1
```

**Comment by amckee** [comment]
Hardcoded to display :0.0, needs detection

https://github.com/symless/synergy-core/blob/426e81fa71d299b2bbbbac64779bb9e004e1520f/src/lib/platform/XWindowsScreen.cpp#L906

**Comment by nlyan** [comment]
This is in sprint for v2.0.7

**Comment by nlyan** [comment]
Fixed in v2.0.7 (not in core)
